python/lsst/afw/geom/xyTransformFactory.py

```diff
@@ -100,7 +101,7 @@ def validate(self):
 def makeRadialXYTransform(config):
     """Make a RadialXYTransform
     """
-    return RadialXYTransform(config.coeffs)
+    return RadialXYTransform(config.coeffs._list)
```

Comment:
Yes, unfortunately exposing the internal `_list` is needed (at least temporarily) because pybind11 doesn't accept all sequence types and just knows about actual Python lists.